mta-sts.md

```diff
@@ -261,7 +220,7 @@ When sending mail via a "smart host"--an intermediate SMTP relay rather than the
 message recipient's server--compliant senders MUST treat the smart host domain
 as the policy domain for the purposes of policy discovery and application.
 
-# Policy Validation
+#Policy Validation
```

Comment:
233 A policy which has been successfully used to deliver mail according to these
 234 constraints is said to be a "validated policy".
This statement may not be required